README.md

```diff
@@ -48,10 +48,15 @@ FastJ is an open-source, Java-based 2D game engine and framework. Originally nam
 ### Template Projects
 Check out these template projects for FastJ! They're the fastest way to jump right into using FastJ.
 
+#### using Gradle
 - Java: https://github.com/lucasstarsz/fastj-java-template
 - Kotlin: https://github.com/lucasstarsz/fastj-kotlin-template
 - Groovy: https://github.com/lucasstarsz/fastj-groovy-template
 
+#### using Maven
```

Comment:
I like the idea, _but_ at the same time I personally planned on keeping FastJ's front-and-center projects as Gradle only.
With that said... there could be a repository containing links to all the different types of templates people come up with.